pkg/workers/sharings/sharing_updates.go

```diff
@@ -269,3 +269,15 @@ func ExtractHostAndScheme(fullURL string) (string, string, error) {
 	}
 	return host, scheme, nil
 }
+
+func isRecipientSide(sharing *Sharing) bool {
```

Comment:
Well, it is normally an impossible scenario, but we never know when a protocol misused could happen, so it actually might be called from the recipient side :)
But if you've been confused, I'll add some comments to clarify.